promql/lex.go

```diff
@@ -571,7 +571,7 @@ func lexValueSequence(l *lexer) stateFn {
 		l.emit(itemBlank)
 	case unicode.IsDigit(r) || (r == '.' && unicode.IsDigit(l.peek())):
 		l.backup()
-		lexNumber(l)
+		lexNumberOrDuration(l)
```

Comment:
Although I see the attractiveness of introducing this, the discussion on the issue didn't seem like agreement at all yet (https://github.com/prometheus/prometheus/issues/419#issuecomment-118858678 and https://github.com/prometheus/prometheus/issues/419#issuecomment-118870286).

I agree that it would be great to resolve this soon, but we should at least wait for @fabxc to make it back from vacation (next week).